### ISSUE TITLE
mostSpecificSubdivision not set when using Twig

### ISSUE THREAD

**Issue by juangiordana** [maxmind/GeoIP2-php#81]
Following the City Example in README.md, I've found that the `mostSpecificSubdivision` (region) is found, but is NULL when used inside Twig templates:

```php
print($record->mostSpecificSubdivision->name . "\n"); // 'Minnesota'
print($record->mostSpecificSubdivision->isoCode . "\n"); // 'MN'

var_dump($record->mostSpecificSubdivision);
```

You can find 'Minnesota' all over the dumped `$record`object, but not the `name` or `isoCode` properties.

```twig
{{ record.mostSpecificSubdivision.isoCode }} // ...
{{ record.mostSpecificSubdivision.isoCode ?: 'N/A' }} // 'N/A'
{{ dump(record.mostSpecificSubdivision) }} // NULL
```

The data is there, just not accessible for some reason.

**Comment by oschwald** [comment]
That is very weird. I don't use Twig. Would if be possible for you to create a runnable test case that we could use when debugging this?

**Comment by juangiordana** [comment]
Here you go:

```sh
$ mkdir ~/test
$ cd ~/test
$ composer require geoip2/geoip2 twig/twig
$ vim test.php
```

```php
<?php
require __DIR__ . '/vendor/autoload.php';

$ipAddress = '128.101.101.200';

$reader = new \GeoIp2\Database\Reader('/usr/share/GeoIP/GeoLite2-City.mmdb');
$record = $reader->city($ipAddress);

$template = <<< 'EOD'
Country is "{{ record.country.name }}", works.
Region is "{{ record.mostSpecificSubdivision.name }}", doesn't work.
Region is "{{ region }}" works.

EOD;

$loader = new Twig_Loader_Array([
    'index.html' => $template
]);
$twig = new Twig_Environment($loader);

echo $twig->render(
    'index.html',
    [
        'record' => $record,
        'region' => $record->mostSpecificSubdivision->name
    ]
);
```

```sh
$ php -f test.php
```

**Comment by oschwald** [comment]
Thanks! That is perfect. That is definitely strange.

**Comment by oschwald** [comment]
This should be fixed in 2.4.4.

**Comment by juangiordana** [comment]
Thanks!
